src/components/Reserve.js

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { createReservation } from '../redux/reservations/reservationSlice';
+
+const Reserve = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const aeroplanesData = useSelector((state) => state.aeroplanes.aeroplanes);
+  const { reservedJet } = useSelector((state) => state.aeroplanes);
+  const userData = JSON.parse(localStorage.getItem('Token')) || {};
+
+  const [data, setData] = useState({
+    username: '',
+    planeName: '',
+    startLocation: '',
+    endLocation: '',
+    startTime: '',
+    endTime: '',
+    reserveDate: '',
+    name: '',
+  });
```

Comment:
- [ ] I cannot reserve planes:

Pry output when trying to reserve a plane:

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/9af61f69-0ea1-4468-bfb2-e739d423c227)

Server response:

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/f8afa7d7-a38b-4dd8-bc01-3da7399df760)

Payload sent to the server:

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/59c3e1de-d7af-4c86-9502-ca71531fcab7)

## Explanation

It seems that the server does not know what "name" is for appointments.

According to schema:

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/8da06f50-d54b-4066-989d-aa5faa4e1d3d)

PSQL:

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/6c89f50d-11d8-416d-ac9d-8ae499ca372c)


Conclusion: There is no "name" column on the appointments table.  Therefore, the server throws an error because it does not know what to do with that `:name` attribute.

Kindly fix this bug so your app works as intended. 👍 

## _note: as a quick fix, I added the column :name to reservations and it worked:_

```
rails generate migration add_name_to_reservations name:string
rails db:migrate
```

### Result

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/bee4afd5-88b9-468b-ab9b-1fc5cca25ea0)

![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/1b3639db-0ed4-402d-81f4-2b1e77c34796)

Reply:
@Meltrust Thanks a lot for pointing this out, it works on our end only because we have the latest db but we would have never noticed this. I truly appreciate it.
We will make the changes and request for review again 😄